middleware/src/middleware.go

```diff
@@ -409,8 +416,8 @@ func (middleware *Middleware) GetHostname() rpcmessages.GetHostnameResponse {
 // EnableTor enables/disables the tor.service and configures bitcoind and lightningd based on the passed ToggleSettingEnable/Disable argument
 // and returns a ErrorResponse indicating if the call was successful.
 func (middleware *Middleware) EnableTor(toggleAction rpcmessages.ToggleSetting) rpcmessages.ErrorResponse {
-	log.Printf("Executing '%s Tor' via the config script.\n", toggleAction)
-	out, err := middleware.runBBBConfigScript([]string{string(toggleAction), "tor"})
+	log.Printf("Executing 'Enable Tor: %v' via the config script.\n", toggleAction)
```

Comment:
Could use [`%t`](https://golang.org/pkg/fmt/#pkg-overview) for boolean types here and in the other Printf's, but not crucial.

Reply:
done